lib/api/completeMultipartUpload.js

```diff
@@ -212,7 +212,8 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 const partSize = Number.parseInt(storedPartsAsObjects[i]
                     .size, 10);
                 if (i < jsonList.Part.length - 1 &&
-                    partSize < 5242880) {
+                    partSize < constants.minimumAllowedPartSize &&
+                    request.relaxedSizeCheckingForTests !== true) {
```

Comment:
I'd rather not complicate the production code to accommodate tests which can be modified.